zeebe/atomix/cluster/src/main/java/io/atomix/raft/snapshot/impl/SnapshotChunkImpl.java

```diff
@@ -45,7 +45,7 @@ public SnapshotChunkImpl(final SnapshotChunk chunk) {
     totalCount = chunk.getTotalCount();
     chunkName = chunk.getChunkName();
     checksum = chunk.getChecksum();
-    snapshotChecksum = chunk.getSnapshotChecksum();
+    snapshotChecksum = 0;
```

Comment:
:wrench: This field can be removed. Instead directly write 0 in `write()`. Also add a comment why it is 0.